### PR TITLE
Update hds to 0.13.3

### PR DESCRIPTION
**Pull request by Sartonon**
## Description

- Update hds to 0.13.3

- Delete redundant stuff now that hds bundle has commonjs

### Related
**[PT-509](https://helsinkisolutionoffice.atlassian.net/browse/PT-509):**

### CODE REVIEW

**Review by jorilindell — COMMENTED** [review]
Looks good.
Maybe you could remove next-transpile-modules from dependencies if it's not is use anymore?

File: src/setupTests.ts [hunk 1]
```diff
@@ -1,4 +1,2 @@
 import '@testing-library/jest-dom';
 import './tests/initI18n';
-
-process.env.SASS_PATH = './src/assets/styles';
```
Comment (jorilindell) [inline src/setupTests.ts:4]: Is there a reason to move SASS_PATH definition back to package.json?

Comment (Sartonon) [reply src/setupTests.ts:4]: Yeah, that one wasn't actually working, it was some cache that made it look like it was working the other day :(. Spend lot of time to yesterday trying to get that env variable working some other way than in package.json but couldn't.

Comment (jorilindell) [reply src/setupTests.ts:4]: Ah, ok. In that case, it's better to go with working solution

**Pull request merged**